lib/generators/react/install_generator.rb

```diff
@@ -110,6 +109,16 @@ def setup_react_webpacker
           create_file(manifest, WEBPACKER_SETUP_UJS)
         end
       end
+
+      private
+
+      def webpack_source_path
+        if Webpacker.respond_to?(:config)
+          Webpacker.config.source_entry_path # Webpacker >3
```

Comment:
it works for me
``` 
Webpacker.config.source_path.join(Webpacker.config.source_entry_path)

```

Reply:
It works..... but is it correct?

Reply:
I think this is correct as-is: `Webpacker.config.source_entry_path`

Reply:
`Webpacker.config.source_entry_path` is the correct way to call this, as this is defined in webpacker.